modules/command/dc/dc-offer-cancel-command.js

```diff
@@ -44,6 +44,11 @@ class DCOfferCancelCommand extends Command {
                 { status: 'CANCELLED' },
                 { where: { import_id: importId, status: { [Op.not]: 'FINALIZED' } } },
             );
+            // update offer.message
+            await Models.offers.update(
+                { message: 'Offer has been cancelled' },
+                { where: { import_id: importId } },
+            );
```

Comment:
Not all, only the last one should be updated. Something like this:
```
            const offerModel = await Models.offers.findOne({
                where: {
                    import_id,
                    status: { [Models.Sequelize.Op.in]: ['FINALIZING', 'FINALIZED'] },
                },
                order: [
                    ['id', 'DESC'],
                ],
            });
```
See EventEmitter.js in `this._on('kad-replication-request', async (request) => {`.